examples/connect/standalone/README.md

```diff
@@ -0,0 +1,70 @@
+# RStudio Connect Standalone
+
+## Example Setup
+
+Add the RStudio Helm charts repository to your Kubernetes cluster:
+
+```sh
+helm repo add rstudio https://helm.rstudio.com
+```
+
+Export the RStudio Connect license key as an environment variable. 
+
+```sh
+export RSC_LICENSE="THIS-IS-THE-LICENSE"
+```
+
+Install the Helm chart using the configuration provided in the [`values.yaml`](./values.yaml)
+file:
+
+```sh
+helm install rstudio/rstudio-connect -f ./values.yaml --set license.key=$RSC_LICENSE --generate-name
+```
+
+## Context
+
+The [`values.yaml`](./values.yaml) file in this folder configures a standalone deployment of the
+RStudio Connect Helm chart. If you want to see the different configurable values
+beyond the ones included in this example execute:
+
+```
+helm show values rstudio/rstudio-connect
+```
+
+The file for this example has four different sections, separated by
+YAML top-level keys, specifying the different areas of configuration:
+
+- #### `sharedStorage`
+
+  - This section configures which storage provisioner to use with the
+    deployment, the size of the storage, which access modes it has, and what
+    will be the mounting point. The example assigns an empty string as that
+    means the deployment will use the cluster's supported storage provisioner.
```

Comment:
```suggestion
    means the deployment will use the cluster's default supported storage provisioner.
```

Add default to clarify? Since clusters can support multiple storage provisioners.